index.d.ts

```diff
@@ -299,10 +299,11 @@ export declare interface TracerOptions {
   };
 
   /**
-   * Whether to load all built-in plugins.
+   * Whether to load all built-in plugins, or an Array of integrations 
+   * to automatically disable when the tracer is initialized e.g. `['express', 'dns']`
    * @default true
    */
-  plugins?: boolean;
+  plugins?: boolean | string[];
```

Comment:
This configuration option should not accept an array as it doesn't have the same meaning as the header.

Reply:
reverted back to original as we're separating out `disabledPlugins` to a separate env var per other comment.